src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_inpaint.py

```diff
@@ -146,7 +146,7 @@ def prepare_mask_and_masked_image(image, mask, height, width):
 
     masked_image = image * (mask < 0.5)
 
-    return mask, masked_image
+    return mask, masked_image, image
```

Comment:
We need to make this function backwards compatible. Could you add an argument `return_image: bool = False` to the function signature so that it's fully  backward compatible?

Reply:
Added this boolean, now returns only `mask, masked_image` by default

Reply:
N.b. set `return_image: bool = True` in the actual new pipeline call 

```
# 5. Preprocess mask and image
mask, masked_image, init_image = prepare_mask_and_masked_image(image, mask_image, height, width, return_image=True)
```